cluster-scope/overlays/nerc-ocp-prod/kustomization.yaml

```diff
@@ -8,6 +8,9 @@ resources:
 - ../../bundles/acct-mgt
 - ../../bundles/patch-operator
 - ../../bundles/xdmod-reader
+- ../../base/core/namespaces/aap
+- ../../base/operators.coreos.com/operatorgroups/aap
+- ../../base/operators.coreos.com/subscriptions/ansible-automation-platform-operator
```

Comment:
We would typically move this collection of files into a *bundle* in `cluster-scope/bundles` (this allows us to activate/deactivate the operator with a single line in our overlay). See e.g. `cluster-scope/bundles/group-sync-operator`.

Reply:
Thanks for the feedback @larsks . I learned some new vim tricks and updated my commit. I also made a bundle for ansible-automation-platform-operator.